bucket/openmw.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This file should use 4 spaces for indentation (like all other manifests).

Reply:
I think this is because I used `scoop create` to generate this manifest. I correcte dit now, thanks.